requirements.txt

```diff
@@ -6,4 +6,5 @@ opencv-python
 pytest
 pynetworktables
 robotpy-cscore
-robotpy-wpimath
\ No newline at end of file
+robotpy-wpimath
+pre-commit
```

Comment:
We're using this requirements file to dictate what will get installed to the coprocessor. I don't think this should go here.
```suggestion
```

Reply:
Jesse tried to clarify this with me, but I didn't even think about it. What do you think is a better place to store it @auscompgeek? Just a secondary requirements file or something different?

Reply:
Given there are multiple ways to install `pre-commit`, e.g. `brew` or other system package manager, I'm not sure this should go in a requirements file at all.

Reply:
It sounds like removing it entirely is your preferred solution here. is that correct?

Reply:
That's probably the pragmatic solution here, yeah.